src/unpack.js

```diff
@@ -141,6 +159,9 @@ function getThriveExecutableName(){
     } else if(os.platform() === "linux"){
 
         return "Thrive";
+    } else if(os.platform() === "darwin"){
+
+        return "Thrive.app/Contents/MacOS/Thrive";
```

Comment:
Now that I had a good look, this is not consistent with the other platforms. This should return "Thrive" and the function that calls this, which is responsible for finding the executable path, should be updated instead to properly find the executable on mac.

Reply:
you mean the function below it: thriveExecutableExistsInFolder(folderToCheck) ?

Reply:
I mean `findBinInRelease`, which when properly updated for mac support should make the entire process of finding the executable work: https://github.com/Revolutionary-Games/Thrive-Launcher/blob/4c1eeb03108e1ec3e3f3f347376f4e77f65728fc/src/thrive_runner.js#L17
the reason that it isn't a hardcoded path is that the thrive folder structure has changed multiple times in the past.